lib/roby/custom_require.rb

```diff
@@ -4,16 +4,17 @@
 module Kernel
     alias syskit_original_require require
     def require(path)
-        # Only deal with paths starting with models/, the other ones are
+        # Only deal with paths starting with #{toplevel_folders}/, the other ones are
         # unambiguous
-        if path =~ /^models\//
+        toplevel_folders = "(models|payloads)"
```

Comment:
Since `payloads` is specific to tidewise (at least for now), don't put it there. Either make it configurable, or (better), just let one require anything at toplevel.

Reply:
changed to require anything at toplevel